Formula/p/pgcopydb.rb

```diff
@@ -0,0 +1,36 @@
+class Pgcopydb < Formula
+  desc "Copy a Postgres database to a target Postgres server"
+  homepage "https://github.com/dimitri/pgcopydb"
+  url "https://github.com/dimitri/pgcopydb/archive/refs/tags/v0.17.tar.gz"
+  sha256 "7ed96f7bbc0a5250f3b73252c3a88d665df9c8101a89109cde774c3505882fdf"
+  license "PostgreSQL"
+  head "https://github.com/dimitri/pgcopydb.git", branch: "main"
+
+  depends_on "sphinx-doc" => :build
+  depends_on "bdw-gc"
+  depends_on "libpq"
+
+  def install
+    system "make", "bin"
+    bin.install "src/bin/pgcopydb/pgcopydb"
+    system "make", "-C", "docs", "man"
+    man1.install Dir["docs/_build/man/*"]
+  end
+
+  def caveats
+    <<~EOS
+      PostgreSQL client CLI tools must be installed for `pgcopydb` to work.
```

Comment:
You depend on them so this seems guaranteed

Reply:
The tooling may not be configured as `libpq` is `keg_only` because it conflicts with `postgres` and in most cases (only not in `postgres@14`) `postgres` is also `keg_only`.

Reply:
They'll definitely be installed. Otherwise it would be a brew bug.